src/widgets/ThemedWrappers.h

```diff
@@ -0,0 +1,114 @@
+/*!********************************************************************
+
+   Audacity: A Digital Audio Editor
+
+   @file ThemedWrappers.h
+
+   @author Vitaly Sverchinsky
+
+   A set of classes which are simplify theme support
+   
+**********************************************************************/
+
+#pragma once
+
+#include <wx/app.h>
+#include "prefs/ThemePrefs.h"
```

Comment:
You include this for `EVT_THEME_CHANGE` and not `PrefsPanel`?

As you might have guessed, that is a usage of wxEvent that I would rather simplify with Observer.

In fact, I have a commit waiting in my big branch of module extractions doing just that.

I should cherry-pick it, and push it onto your branch, and let you merge it with this branch.